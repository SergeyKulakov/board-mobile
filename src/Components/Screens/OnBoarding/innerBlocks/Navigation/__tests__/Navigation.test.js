import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { Navigation as Component } from '../index'

describe('OnBoarding.Navigation', () => {
  const defaultProps = {
    activeScreen: 1,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
