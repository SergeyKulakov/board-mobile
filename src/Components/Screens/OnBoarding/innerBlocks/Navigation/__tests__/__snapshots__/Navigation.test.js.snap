// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnBoarding.Navigation render snapshot 1`] = `
<Styled(View)>
  <Styled(View)
    active={false}
  />
  <Styled(View)
    active={true}
  />
  <Styled(View)
    active={false}
  />
  <Styled(View)
    active={false}
  />
</Styled(View)>
`;
